src/SDKS/erc20/index.ts

```diff
@@ -232,7 +256,7 @@ export namespace CryptoWallet.SDKS.ERC20 {
      * Gets the balance of the ERC20 token on a users ethereum account
      * @param erc20Wallet
      */
-    getBalance(
+    public getBalance(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Declaration of public instance method not allowed after declaration of private instance method. Instead, this should come after public instance fields.](https://app.codacy.com/app/atlascity/cryptowallet-js/pullRequest?prid=3649440)